app/controllers/internal/configs_controller.rb

```diff
@@ -47,6 +47,6 @@ def clean_up_params
   end
 
   def bust_relevant_caches
-    CacheBuster.bust("/api/tags/onboarding") # Needs to change when suggested_tags is edited
+    CacheBuster.bust("/tags/onboarding") # Needs to changed when suggested_tags is edited
```

Comment:
I think this comment was originally fine. Or if we wanted to change it, how about something like:
`# Needs to be changed when suggested_tags is edited`

Reply:
It was, somehow I read it as "need to be" anyway. 🤦‍♂